Underscore/USConstants.h

```diff
@@ -32,6 +32,7 @@ typedef id   (^UnderscoreReduceBlock)(id memo, id obj);
 typedef void (^UnderscoreArrayIteratorBlock)(id obj);
 typedef id   (^UnderscoreArrayMapBlock)(id obj);
 typedef id   (^UnderscoreArrayZipWithBlock)(id objectFromFirstArray, id objectFromSecondArray);
+typedef void (^UnderscoreIntegerIteratorBlock)(NSInteger n);
```

Comment:
Thank you for review! :sushi: 

I think this block is not restricted positive number, it is a more general block for Integer. 
(Or, if it restrict to positive number, I think that should be renamed to `UnderscoreUIntegerIteratorBlock` or `UnderscoreTimesIteratorBlock`.)

And to obviate negative case, If anything,  I think should make [this line](https://github.com/akitsukada/Underscore.m/blob/67db906558bb563ed791f5206c0ac378acf7befa/Underscore/Times/Underscore%2BTimes.h#L31) a `NSUInteger`.
But if `NSInteger` value is passed to `NSUInteger` argument, It will be treated as very big positive value (and loops many times) without compile errors.
So I think NSInteger is better than NSUInteger, because negative value is  obviated by `for` [statement condition(`count < times`)](https://github.com/akitsukada/Underscore.m/blob/67db906558bb563ed791f5206c0ac378acf7befa/Underscore/Times/Underscore%2BTimes.m#L34) as negative value, and it is safety.
Is it better that use `NSUInteger`? I'm wondering...